en/activity-standard/iati-activities/iati-activity/default-aid-type.rst

```diff
@@ -2,14 +2,25 @@ Example Usage
 ~~~~~~~~~~~~~
 Example ``default-aid-type``  for an ``iati-activity``.
 
-| The ``@code`` attribute declares a valid code (*A01*) from the *AidType* codelist.
+| The ``@code`` attribute declares a valid code from the specified codelist.
 | The ``@vocabulary`` attribute declares a valid code (*1*) from the *AidTypeVocabulary* codelist.
 
+.. code-block:: xml
+
+	<default-aid-type code="A01" vocabulary="1" />
+
+
+The default-aid-type element can be repeated in any iati-activity.
+
 .. literalinclude:: ../../activity-standard-example-annotated.xml
 	:language: xml
 	:start-after: <!--default-aid-type starts-->
 	:end-before: <!--default-aid-type ends-->
 
+The vocabulary is used to declare which AidType list is in use. If this is omitted, then IATI assumes an AidType of 1 (OECD DAC).
```

Comment:
> If this is omitted, then IATI assumes an AidType of 1

This is about Aid Type Vocabulary, not Aid Type. If omitted, an **Aid Type Vocabulary** of 1 is assumed.

Reply:
Thanks for spotting this @andylolz